internal/commands/pull/command.go

```diff
@@ -258,17 +258,22 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		ui.Print(terminal.NewDebugLog("Fetched hosting assets"))
 	}
 
-	successfulTemplateWrites := make([]string, 0, len(clientTemplates))
+	successfulTemplateWrites := make([]interface{}, 0, len(clientTemplates))
```

Comment:
rather than manage the `"  "` yourself in the `"  Navigate..."` message, you could initialize this to be:
```suggestion
	successfulTemplateWrites := make([]interface{}, 0, len(clientTemplates)+1)
```
and then simply append the "Navigate string" as one last item to get the same effect here